config/guestosconfig/known-modules.json

```diff
@@ -0,0 +1,69 @@
+[
+  { "moduleName": "xt_MASQUERADE"},
+  { "moduleName": "xt_addrtype"},
+  { "moduleName": "iptable_nat"},
+  { "moduleName": "nf_nat"},
+  { "moduleName": "br_netfilter"},
+  { "moduleName": "ip6table_filter"},
+  { "moduleName": "ip6_tables"},
+  { "moduleName": "aesni_intel"},
+  { "moduleName": "glue_helper"},
+  { "moduleName": "crypto_simd"},
+  { "moduleName": "cryptd"},
+  { "moduleName": "virtio_balloon"},
+  { "moduleName": "loadpin_trigger"},
+  { "moduleName":"ip6table_filter"},
+  { "moduleName":"ip6_tables"},
+  { "moduleName":"iptable_filter"},
+  { "moduleName":"bpfilter"},
+  { "moduleName":"nls_iso8859_1"},
+  { "moduleName":"intel_rapl_msr"},
+  { "moduleName":"intel_rapl_common"},
+  { "moduleName":"sb_edac"},
+  { "moduleName":"rapl"},
+  { "moduleName":"input_leds"},
+  { "moduleName":"serio_raw"},
+  { "moduleName":"pvpanic"},
+  { "moduleName":"mac_hid"},
+  { "moduleName":"sch_fq_codel"},
+  { "moduleName":"ib_iser"},
+  { "moduleName":"rdma_cm"},
+  { "moduleName":"iw_cm"},
+  { "moduleName":"ib_cm"},
+  { "moduleName":"ib_core"},
+  { "moduleName":"iscsi_tcp"},
+  { "moduleName":"libiscsi_tcp"},
+  { "moduleName":"libiscsi"},
+  { "moduleName":"scsi_transport_iscsi"},
+  { "moduleName":"virtio_rng"},
+  { "moduleName":"ip_tables"},
+  { "moduleName":"x_tables"},
+  { "moduleName":"autofs4"},
+  { "moduleName":"btrfs"},
+  { "moduleName":"zstd_compress"},
+  { "moduleName":"raid10"},
+  { "moduleName":"raid456"},
+  { "moduleName":"async_raid6_recov"},
+  { "moduleName":"async_memcpy"},
+  { "moduleName":"async_pq"},
+  { "moduleName":"async_xor"},
+  { "moduleName":"async_tx"},
+  { "moduleName":"xor"},
+  { "moduleName":"raid6_pq"},
+  { "moduleName":"raid1"},
+  { "moduleName":"raid0"},
+  { "moduleName":"multipath"},
+  { "moduleName":"linear"},
+  { "moduleName":"crct10dif_pclmul"},
+  { "moduleName":"crc32_pclmul"},
+  { "moduleName":"ghash_clmulni_intel"},
+  { "moduleName":"aesni_intel"},
+  { "moduleName":"crypto_simd"},
+  { "moduleName":"cryptd"},
+  { "moduleName":"glue_helper"},
+  { "moduleName":"psmouse"},
+  { "moduleName":"virtio_net"},
+  { "moduleName":"net_failover"},
+  { "moduleName": "failover"},
+  { "moduleName":"i2c_piix4"}
+]
```

Comment:
Missing new line at the end of the file.

Reply:
done